src/hooks/useSyntheticsTest.ts

```diff
@@ -106,3 +106,9 @@ export function useSyntheticsTest(steps: Steps): ITestContext {
     setResult,
   };
 }
+
+export type RunJourneyOptions = {
```

Comment:
Why is this defined here?

Reply:
I _think_ the intention was to keep the type definition close to the function that requires this type. But it does look weird! I will move it under `shared`.